### ISSUE TITLE
.drone.yml:18-22: Rm npm step. Currently npm fails, when...

### ISSUE THREAD

**Issue by 0pdd** [fidals/stroyprombeton#280]
The puzzle `457-03c18b41` from #457 has to be resolved:

https://github.com/fidals/stroyprombeton/blob/8834e7524b9abeb55662ea33ab2a6c5b8ac918a9/.drone.yml#L18-L22

The puzzle was created by duker33 on 02-Aug-18. 

Estimate: 60 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by duker33** [comment]
blocked by https://github.com/fidals/shopelectro/issues/477
This task contains problem itself, i suppose. Problem is outside of stale images.
